auth-web/src/components/auth/EntityManagement.vue

```diff
@@ -43,16 +43,6 @@
               </v-list-item>
             </v-list>
           </v-menu>
-          <v-btn
```

Comment:
I dont know the logic here..but hope its fine to remove this code

Reply:
Yes its fine, as the flag is to toggle the button in the else part